src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -84,13 +84,15 @@ final class OnboardingInfoViewController: UIViewController {
 
 	private var onboardingInfos = OnboardingInfo.testData()
 	private var exposureManagerActivated = false
+	private var pageSetupDone = [OnboardingPageType: Bool]()
```

Comment:
Wouldn't it be enough to have just a bool? The OnboardingInfoViewController is initialized only with one pageType. And the status "done" is independent of the type.

Reply:
done